shippable.yml

```diff
@@ -208,6 +208,44 @@ jobs:
       script:
         - shipctl put_resource_state_multi $JOB_NAME "versionName=$REPO_COMMIT" "FILE_REPO_COMMIT_SHA=$REPO_COMMIT" "S3_BUCKET=$S3_BUCKET_BINARY_DIR" "S3_FILENAME=$BINARY_TAR" "ARCHITECTURE=$RCHITECTURE" "OS=$OS"
 
+  - name: u16_reqExec_aarch32_pack
+    type: runSh
+    runtime:
+      nodePool: aarch32_u1604_cus_01
+    triggerMode: parallel
+    dependencyMode: strict
+    steps:
+      - IN: reqExec_runCI
+      - IN: reqExec_sh_repo
+        switch: off
+      - IN: shipit_bits_access_cli
+        switch: off
+      - TASK:
+          name: u16_aarch32_reqexec_pack
+          runtime:
+            options:
+              imageName: drydock/aarch32_u16
+              imageTag: master
+              env:
+                - ARCHITECTURE: "aarch32"
+                - OS: "Ubuntu_16.04"
+                - ARTIFACTS_BUCKET: "s3://shippable-artifacts/reqExec"
+                - REL_VER: "master"
+                - BINARY_DIR: "/tmp/reqExec"
+          script:
+            - pushd $(shipctl get_resource_state "reqExec_sh_repo")
+            - REPO_COMMIT=$(shipctl get_resource_version_key "reqExec_sh_repo" "shaData.commitSha")
+            - BINARY_TAR="reqExec-$REL_VER-$ARCHITECTURE-$OS.tar.gz"
+            - PACK_SCRIPT="./package/$ARCHITECTURE/$OS/package.sh"
+            - S3_BUCKET_BINARY_DIR="$ARTIFACTS_BUCKET/$REL_VER/"
+            - $PACK_SCRIPT
+            - mkdir -p $BINARY_DIR && cp -r dist $BINARY_DIR
```

Comment:
You'll first need to `rm` $BINARY_DIR if it exists or you'll end up having lots of weirdness by copying in to an existing directory..